### PR TITLE
Ch 3384: Add ability to specify pages for personalize elements variation sets

### PR DESCRIPTION
**Pull request by eshtadc**
Also added link to edit javascript webpage element sets.

### CODE REVIEW

File: acquia_lift.admin.unibar.inc [hunk 1]
```diff
@@ -209,6 +209,7 @@ function acquia_lift_element_variation_details_form_submit($form, &$form_state)
 
     module_load_include('inc', 'personalize_elements', 'personalize_elements.admin');
     $option_set = personalize_elements_convert_form_values_to_option_set($values);
+    $option_set->preview_link = $values['pages'];
```
Comment (katbailey) [inline acquia_lift.admin.unibar.inc:214]: Surely `$values['pages']` could list multiple pages?

Comment (eshtadc) [inline acquia_lift.admin.unibar.inc:214]: Not at that point.  This is only when creating a new personalize elements variation set from the unibar.  The unibar populates the pages field with the current url using JavaScript.  The user can't touch it.

File: acquia_lift.admin.wizard.inc [hunk 2]
```diff
@@ -1844,10 +1885,7 @@ function _acquia_lift_personalize_campaign_wizard_block_single_save($block_conte
 function _acquia_lift_personalize_campaign_wizard_element_single_save($values, $option_set) {
   // Make sure to start with the latest option set information.
   $option_set = personalize_option_set_load($option_set->osid);
-  $option_set->label = $values['label'];
-  foreach ($option_set->options as &$option) {
-    $option['option_label'] = $values['options'][$option['option_id']]['option_label'];
-  }
```
Comment (katbailey) [inline acquia_lift.admin.wizard.inc:1850]: Why is this being deleted? Seems unrelated.

File: acquia_lift.admin.wizard.inc [hunk 3]
```diff
@@ -1868,6 +1906,11 @@ function _acquia_lift_personalize_campaign_wizard_variations_single_save_general
     $option_label = isset($values['content']['title']) ? $values['content']['title'] : $values['content']['label'];
     $option_set->label = $option_label;
     $values['advanced']['label'] = $option_label;
+    foreach ($option_set->options as &$option) {
+      if (isset($values['content']['options'][$option['option_id']]['option_label'])) {
+        $option['option_label'] = $values['content']['options'][$option['option_id']]['option_label'];
+      }
+    }
```
Comment (katbailey) [inline acquia_lift.admin.wizard.inc:1913]: Oh sorry, it was just moved :)

File: acquia_lift.admin.unibar.inc (hunk 1 above)
Comment (katbailey) [inline acquia_lift.admin.unibar.inc:214]: Ah cool
